superset/assets/spec/javascripts/dashboard/Controls_spec.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { describe, it, beforeEach } from 'mocha';
+import { expect } from 'chai';
+
+import Controls from '../../../javascripts/dashboard/components/Controls';
+import { dashboardData } from './fixtures';
+
+describe('Controls', () => {
+  let wrapper;
+  const mockedProps = {
+    dashboard: dashboardData,
+  };
+  mockedProps.dashboard.css = 'foo';
+
+  beforeEach(() => {
+    wrapper = mount(<Controls {...mockedProps} />);
+  });
+
+  it('is a valid element', () => {
+    expect(React.isValidElement(<Controls {...mockedProps} />)).to.equal(true);
+  });
+
+  it('sets the state to cssTemplates on componentWillMount', () => {
```

Comment:
the goal is to mock the AJAX request and have the mock return what would be expected, and then assert that the state is updated once that happens.